File: src/main/java/de/zerotask/minecraft/tooltweaks/server/ServerProxy.java
package de.zerotask.minecraft.tooltweaks.server;

import de.zerotask.minecraft.tooltweaks.common.CommonProxy;
import de.zerotask.minecraft.tooltweaks.common.network.PacketHandler;
import net.minecraftforge.common.MinecraftForge;

public class ServerProxy extends CommonProxy {

    @Override
    public void init() {
        super.init();
        MinecraftForge.EVENT_BUS.register(new PacketHandler());
    }
}

File: src/main/kotlin/de/zerotask/minecraft/tooltweaks/server/ServerProxy.kt
package de.zerotask.minecraft.tooltweaks.server

import de.zerotask.minecraft.tooltweaks.common.CommonProxy
import de.zerotask.minecraft.tooltweaks.common.network.PacketHandler
import net.minecraftforge.common.MinecraftForge

class ServerProxy: CommonProxy() {

    override fun init() {
        super.init()
        MinecraftForge.EVENT_BUS.register(PacketHandler())
    }
}